subcommittees/mentoring/README.md

```diff
@@ -1,73 +1,100 @@
+### Mentoring Program
 
-To see the schedule for the 2015 retreat, visit: http://swcarpentry.github.io/instructor-retreat-2015/
+This program supports instructors who are new to our community by matching them with a personal Mentor. Mentors will guide small groups (~4-5) of Mentees, holding meetings for their Mentees to do things like practice teaching, ask questions about workshop logistics, and brush up on lesson content, depending on their needs and interests. 
 
-### Archived: Pre- and Post- Workshop Debriefing Session
+See [this page](https://github.com/datacarpentry/mentoring-program/blob/master/program-outline.md) for a full outline of the program.  
 
-Before launching the Instructor Discussion Sessions, we used to host separate [pre- and post-workshop debriefing sessions](http://software-carpentry.org/blog/categories/#debriefing). We are grateful to all the hosts and participants who attended these sessions.
+### Mentoring Committee Meetings
 
-## Mailing List
+The mentoring committee itself meets regularly to evaluate current activities, and propose
+new activities.  Minutes of past meetings are available [here][minutes].  All community 
+members are welcome to attend mentoring committee meetings.  Similarly, we welcome 
+anyone who is interested in joining up and helping out!  
 
-The best way to contact this subcommittee is via the [mailing
-list][mailing-list], which is archived [here][mailing-list-archives].
+To attend our next meeting, visit this link: http://pad.software-carpentry.org/scf-mentoring
 
-## Outreach
 
-We [blog][] about our ongoing and proposed activities.  Past blog
-posts are available [here][blog-archives].
+## How Can I Participate?  
 
-## Meetings
+Any Software or Data Carpentry community member is welcome to participate in 
+Mentoring committee activities and/or join the committee itself!  Some different 
+options for getting involved are given below.  
 
-We have occasional [meetings][].  Minutes of past meetings are
-available [here](minutes).
+I would like to: 
 
-## Join Us!
+- join a small group to receive mentoring from experienced instructors OR
+- mentor new instructors
+	- Sign up for the mentoring program by contacting Erin Becker (ebecker@datacarpentry.org)
+- discuss workshop experiences with other instructors OR
+- ask questions of other instructors as I prepare to teach
+	- Attend an instructor discussion session by signing up on this etherpad: http://pad.software-carpentry.org/instructor-discussion
+- help lead discussion sessions among instructors
+	- Join the [discussion host mailing list][host-mailing-list] and ign up to host, co-host, or observe 
+	a discussion on this etherpad: http://pad.software-carpentry.org/instructor-discussion
+- support current mentoring activities or initiate new ones
```

Comment:
Add a trailing “OR”?  (that seems to be your pattern from previous entries in this list)